etc/test-helper.js

```diff
@@ -46,16 +46,16 @@ function verifyDocumentAccepted(doc, oldDoc, expectedChannels) {
   }
 }
 
-function verifyDocumentCreated(doc, oldDoc) {
-  verifyDocumentAccepted(doc, oldDoc, 'add');
+function verifyDocumentCreated(doc, expectedChannels) {
```

Comment:
For the sake of convenience, there's no need for the caller to include the old document when creating a new document.